src/index.html

```diff
@@ -4,9 +4,44 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/style.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
```

Comment:
You do not need to create additional `div`, the `container` class can be specified for the `body`.